docs/resources/kubernetes_manifest.md

```diff
@@ -22,10 +22,16 @@ Once applied, the `object` attribute reflects the state of the resource as retur
 
 - **object** (Dynamic, Optional) The resulting resource state, as returned by the API server after applying the desired state from `manifest`.
 - **wait_for** (Object, Optional) (see [below for nested schema](#nestedatt--wait_for))
+- **timeouts** (Object, Optional) (see [below for nested schema](#nestedatt--timeouts))
```

Comment:
This is technically defined as map in schema, rather than object. Not sure how much of a difference that makes to users, though.